src/Host/Client/Impl/Transports/IWebSocketClient.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.WebSockets;
+using System.Threading.Tasks;
+using System.Text;
+using System.Threading;
+
+namespace Microsoft.R.Host.Client {
+    public interface IWebSocketClient {
```

Comment:
yes. eventually. There are a few authentication related questions regarding wss (web socket secure).

---
In reply to: [111821736](https://github.com/Microsoft/RTVS/pull/3419#discussion_r111821736) [](ancestors = 111821736)